debugging/book-library/script.js

```diff
@@ -75,12 +79,7 @@ function render() {
     changeBut.id = i;
     changeBut.className = "btn btn-success";
     wasReadCell.appendChild(changeBut);
-    let readStatus = "";
-    if (myLibrary[i].check == false) {
-      readStatus = "Yes";
-    } else {
-      readStatus = "No";
-    }
+    let readStatus = myLibrary[i].check ? "Read" : "Unread";
```

Comment:
It would be nice to see  button as red  if book  is marked as unread to increase UI&UX